docs/team/nehcuy.md

```diff
@@ -3,26 +3,82 @@ layout: page
 title: Yuchen's Project Portfolio Page
 ---
 
-### Project: checkUp
+## Project: checkUp
 
-"checkUp" is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+"checkUp" is a desktop address book application used for teaching Software Engineering principles.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java,
+and has about 15 kLoC.
```

Comment:
In a future update, you might want to reword this from the stock lines given in the skeleton draft.

Reply:
Fixed. Thanks for the catch!